mantle/kola/harness.go

```diff
@@ -152,6 +160,12 @@ var (
 	}
 )
 
+// kolaExtBinDataDir is where data will be stored
+const kolaExtBinDataDir = "/var/opt/kola/extdata"
```

Comment:
Should this be suffixed by the test name? Would make it easier to run multiple tests in parallel in the future.

Reply:
Running tests in parallel on the same machine conflicts with the idea of destructive tests (which, as far as kola knows all tests are today).

One can already today drop multiple executables in the same dir, have them share the same data, and do "internal parallelization" at least.

Reply:
Technically the `NativeFuncs` are considered non-destructive and run on the same machine but they get reported as sub-tests. I had a PR forever ago that never ended up landing which worked at adding non-destructive tests (https://github.com/coreos/mantle/pull/697) if someone ever wants to pick that up and re-work it to fit modern mantle.